src/features/campaigns/layout/ProjHomeLayout.tsx

```diff
@@ -0,0 +1,148 @@
+import { ZetkinCampaign, ZetkinOrganization } from 'utils/types/zetkin';
+import ZUILogoLoadingIndicator from 'zui/ZUILogoLoadingIndicator';
+import ZUIButton from 'zui/components/ZUIButton';
+import ZUIOldAvatar from 'zui/ZUIAvatar';
+import ZUIPersonAvatar from 'zui/components/ZUIPersonAvatar';
+import ZUIText from 'zui/components/ZUIText';
+import ZUILink from 'zui/components/ZUILink';
+import useMembership from '../../organizations/hooks/useMembership';
+import useConnectOrg from '../../organizations/hooks/useConnectOrg';
+
+type Props = {
+  children: ReactNode;
+  org: ZetkinOrganization;
+  proj: ZetkinCampaign;
+};
+
+const ProjHomeLayout: FC<Props> = ({ children, org, proj }) => {
+  const messages = useMessages(messageIds);
+  const env = useEnv();
+
+  const user = useUser();
+  const membership = useMembership(org.id).data;
+
+  const { connectOrg } = useConnectOrg(org.id);
+
+  return (
+    <Box
+      sx={{
+        marginX: 'auto',
+        maxWidth: 960,
+      }}
+    >
+      <Box
+        sx={(theme) => ({
+          bgcolor: theme.palette.grey[100],
+          display: 'flex',
+          flexDirection: 'column',
+        })}
+      >
+        <Box sx={{ mb: 6, minHeight: 40, mt: 2, mx: 2, opacity: 0.7 }}>
+          {org.parent && (
+            <NextLink href={`/o/${org.parent.id}`} passHref>
+              <ZUIButton label={org.parent.title} startIcon={NorthWest} />
+            </NextLink>
+          )}
+        </Box>
+        <Box
+          sx={{
+            alignItems: 'center',
+            display: 'flex',
+            justifyContent: 'space-between',
+            mx: 2,
+          }}
+        >
+          <Box sx={{ alignItems: 'center', display: 'flex', gap: 1 }}>
+            <ZUIOldAvatar size="md" url={`/api/orgs/${org.id}/avatar`} />
+            <ZUIText variant="headingLg">{org.title}</ZUIText>
+
+            {user && !membership && (
+              <ZUIButton
+                label={messages.home.header.connect()}
+                onClick={() => connectOrg()}
+              />
+            )}
+            {!user && (
+              <ZUIButton
+                href={`/login?redirect=${encodeURIComponent(`/o/${org.id}`)}`}
+                label={messages.home.header.login()}
+              />
+            )}
+          </Box>
+          {user && (
+            <NextLink href="/my">
+              <ZUIPersonAvatar
+                firstName={user.first_name}
+                id={user.id}
+                lastName={user.last_name}
+              />
+            </NextLink>
+          )}
+        </Box>
+        <Box
+          sx={{
+            alignItems: 'center',
+            display: 'flex',
+            justifyContent: 'space-between',
+            mx: 2,
+            my: 1,
+          }}
+        >
+          <Box sx={{ alignItems: 'center', display: 'flex', gap: 1 }}>
+            <ZUIText variant="headingLg">{proj.title}</ZUIText>
+            <ZUIText>{proj.info_text}</ZUIText>
```

Comment:
Since a project will not always have an info text, i think for a little added elegance and clarity of the fact we can do a lil thing like this on line 105  `{proj.info_text? <ZUIText>{proj.info_text}</ZUIText>}`